### PR TITLE
OSD-6981, allow SREP to get common resources from other namespaces

### PR DESCRIPTION
**Pull request by qhua948**
https://issues.redhat.com/browse/OSD-6981

### CODE REVIEW

File: deploy/backplane/srep/10-srep-readers-project.ClusterRole.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - daemonsets
+  - roles
+  - deployments
+  - replicasets
+  - statefulsets
+  - jobs
+  - cronjobs
+  verbs:
+  - get
+  - list
```
Comment (2uasimojo) [inline deploy/backplane/srep/10-srep-readers-project.ClusterRole.yml:21]: I think `watch` would also be good to include.

**Review by 2uasimojo — COMMENTED** [review]
Thanks for this.

File: deploy/backplane/srep/10-srep-readers-project.ClusterRole.yml (hunk 1 above)
Comment (qhua948) [reply deploy/backplane/srep/10-srep-readers-project.ClusterRole.yml:21]: :+1:

**Pull request merged**